src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1351,7 +1351,8 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
     unsigned tx_cq_size;
 
     ucs_trace_func("");
-
+    //TODO - check with Yossi
+    config->super.super.inl[UCT_IB_DIR_RX] = 0;
```

Comment:
set to 0 here:
https://github.com/openucx/ucx/blob/45114cc741d7087d6d20217bb4b22060c742e099/src/uct/ib/rc/base/rc_iface.c#L31